<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TasksDeleteWindow</class>
 <widget class="QDialog" name="TasksDeleteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>880</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="DeleteTaskGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>840</width>
     <height>290</height>
    </rect>
   </property>
   <property name="title">
    <string>Delete Task</string>
   </property>
   <widget class="QGroupBox" name="EventsGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>500</width>
      <height>210</height>
     </rect>
    </property>
    <property name="title">
     <string>Events</string>
    </property>
    <widget class="QTableView" name="EventsTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>480</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>800</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout">
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteTaskButton">
       <property name="text">
        <string>Delete Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="TasksForTheEventGroupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>290</width>
      <height>210</height>
     </rect>
    </property>
    <property name="title">
     <string>Tasks For The Event </string>
    </property>
    <widget class="QTableView" name="TasksForTheEventTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>270</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
